terraform-tests/terraform_tests_suite_test.go

```diff
@@ -56,3 +56,12 @@ func getAWSRegionFromCSBDefaults() string {
 	}
 	return ""
 }
+
+func getAWSVPCID() string {
+	envVPCID, defined := os.LookupEnv("AWS_PAS_VPC_ID")
+	if defined {
+		return envVPCID
+	}
+
+	return "vpc-72464617"
```

Comment:
Is there a way to look this up? I think this ID is tied to the account of the team maintaining this brokerpak. Being open source, it would be nice if the tests could run in a different AWS account. Even if the answer is to require it to be passed in as an environment variable.

Reply:
I realise that this is no worse that the previous implementation...

Reply:
We can either:
- hardcode the id in the AWS-CSB pipeline
- create [an equivalent PR as this one ](https://github.com/pivotal/cloud-service-broker-ci/pull/630)for AWS-CSB pipeline to enforce terraform-tests to use the default VPC